diff --git a/node_modules/@angular-builders/jest/dist/jest-config/setup.js b/node_modules/@angular-builders/jest/dist/jest-config/setup.js
index 73bd1a8..f07d9bd 100644
--- a/node_modules/@angular-builders/jest/dist/jest-config/setup.js
+++ b/node_modules/@angular-builders/jest/dist/jest-config/setup.js
@@ -1,5 +1,5 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-const zone_1 = require("jest-preset-angular/setup-env/zone");
-(0, zone_1.setupZoneTestEnv)();
+const zone_1 = require("jest-preset-angular/setup-env/zoneless");
+(0, zone_1.setupZonelessTestEnv)();
 //# sourceMappingURL=setup.js.map
